jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -4,172 +4,199 @@
 
 /** Status React component used to display Juju status. */
 class Status extends React.Component {
+  /**
```

Comment:
I am not sure a class should start with the least interesting non-methods [shrug].
Why these can't be just functions? They really don't need the Status state.